std/node/util.ts

```diff
@@ -1,3 +1,4 @@
+export * as types from "./_util/_util_types.ts";
```

Comment:
```suggestion
import * as types from "./_util/_util_types.ts";
export { types };
```

current lint configuration complains about this.

Reply:
Thanks for the suggestion. I have fixed it and the build is now green.